packages/lib-classifier/src/store/ResourceStore.js

```diff
@@ -14,10 +14,11 @@ const ResourceStore = types
   .actions(self => ({
     fetchResource: flow(function * fetchResource (id) {
       const client = getRoot(self).client.panoptes
+      const host = getRoot(self).apiHost
       const { type } = self
       self.loadingState = asyncStates.loading
       try {
-        const response = yield client.get(`/${type}/${id}`)
+        const response = yield client.get(`/${type}/${id}`, undefined, undefined, host)
```

Comment:
This is missing an auth token for protected resources too.

Reply:
Auth hasn't been hooked up in the classifier AFAIK

Reply:
Same again